tests/test_currency_network_erc165.py

```diff
@@ -0,0 +1,41 @@
+#! pytest
+import pytest
+
+from tldeploy.core import deploy_network
+from .conftest import EXPIRATION_TIME
+
+
+NETWORK_SETTING = {
+    "name": "TestCoin",
+    "symbol": "T",
+    "decimals": 6,
+    "fee_divisor": 0,
+    "default_interest_rate": 0,
+    "custom_interests": False,
+    "currency_network_contract_name": "TestCurrencyNetwork",
+    "set_account_enabled": True,
+    "expiration_time": EXPIRATION_TIME,
+}
+
+ERC165_INTERFACE_ID = "0x01ffc9a7"
+CURRENCY_NETWORK_INTERFACE_ID = "0xdcd45f8a"
```

Comment:
How did you get that?
I would rather it is not hard coded, as we need to check that it is not `0xffffffff` and it is not obvious.

Reply:
I calculated it from the function signatures. Put the calculation in the test, so it's more obvious.